src/shoobx/mocks3/models.py

```diff
@@ -35,6 +35,9 @@ def _decode_name(name):
     return name.replace("__sl__", "/")
 
 
+ACCOUNT_ID = "12345678910"
```

Comment:
A comment with the link to http://docs.getmoto.org/en/latest/docs/multi_account.html would be nice.

Reply:
Or a variable might be called `MOTO_DEFAULT_ACCOUNT_ID`.